HISTORY.md

```diff
@@ -1,5 +1,19 @@
 # History
 
+## v0.15.0 - 2024-07-12
+
+This release adds support for NumPy 2.0!
```

Comment:
We can also include the fact that the visualizations now work for just real or synthetic data